src/reference/config/writing-platform-configs.rst

```diff
@@ -49,277 +49,282 @@ Why Were Platforms Introduced?
+   You have a cluster where you can submit jobs from the Cylc scheduler host
+   using PBS.
 
-.. admonition:: Scenario
+The ``localhost`` platform is the Cylc Scheduler host, as configured in
+:cylc:conf:`global.cylc[scheduler][run hosts]available`. This is the host that
+the workflow will start on. For more information, see
+:ref:`Platform Configuration<PlatformConfig>`.
 
-   Users want to run background jobs on a single server,
-   which doesn't share a file system with the workflow host.
+Our platform ``pbs_cluster`` shares this ``localhost`` host and setting the
+install target to ``localhost`` ensures that Cylc knows this platform does not
+require remote initialization.
 
 .. code-block:: cylc
    :caption: part of a ``global.cylc`` config file
 
    [platforms]
-       [[myhost]]
-           hosts = myhost
-           install target = myhost
+       # The localhost platform is available by default
+       # [[localhost]]
+       #     hosts = localhost
+       #     install target = localhost
+       [[pbs_cluster]]
+           hosts = localhost
+           job runner = pbs
+           install target = localhost
 
+Our Cylc scheduler does not have a job runner defined. Any job submitted to
+this ``localhost`` platform will run as a background job. Users can now set 
+:cylc:conf:`flow.cylc[runtime][<namespace>]platform` = ``pbs_cluster`` to run
+pbs jobs.
 
-Cluster with Multiple Login Nodes
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+.. note::
 
-- **Platforms can group multiple hosts together.**
+   Both ``hosts`` and ``install target`` default to the platform name.
+
+Multiple Platforms Sharing File System with Cylc Scheduler
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- **Platform names can be defined as regular expressions.**
 
 .. admonition:: Scenario
 
-   You have a cluster where users submit to a single Slurm job queue from
-   either of a pair of identical login nodes which share a file system:
+   Everyone in your organization has a computer called ``desktopNNN``,
+   all with a file system shared with the scheduler host. Many users
+   will want their desktop set up as a platform to run small jobs.
+
+In this scenario, Cylc does not need to install files on the desktop, since
+required files which are on the scheduler host will be accessible on the
+desktop. From Cylc's point of view, the desktop and scheduler hosts are
+considered different platforms but must share an :term:`install target`.
+Cylc needs to be told that these platforms share an install target and so we
+configure this using the designated configuration item:
+:cylc:conf:`global.cylc[platforms][<platform name>]install target`.
+Cylc has optional configuration ``[[[meta]]]`` to add a description of the
+platform, this may be helpful to use, we will add a platform description to our
+desktop platform.
+The following platform definition is simplified, taking advantage of defaults
```

Comment:
- I think it's worth saying that it's the platform config (not Cylc) which has the `meta` section, since both workflows and tasks also have a meta section.

- I think paragraph breaks for readability.

- I've tried to make it clearer that metadata is there to help users. (In the workflow and task it can have a functional purpose.)

```suggestion
:cylc:conf:`global.cylc[platforms][<platform name>]install target`.

:cylc:conf:`global.cylc[platforms][<platform name>] has optional configuration 
``[[[meta]]]`` which users can view with ``cylc config --platforms``. We will add
a description designed to help users in this example.

The following platform definition is simplified, taking advantage of defaults
```